### ISSUE TITLE
Notifications in Dark Mode hard to read

### ISSUE THREAD

**Issue by simonja7** [mastodon/mastodon#21070]
### Steps to reproduce the problem

1. Set Dark Mode
2. Select Notifications (All)
3. Hard to read
4. Note the difference between a followed notification and a follower notification
...
<img width="784" alt="Screen Shot 2022-11-19 at 4 12 28 PM" src="https://user-images.githubusercontent.com/118582994/202831528-3989babb-61ec-44a5-af32-4edd8b91e214.png">

### Expected behaviour

Notifications should be easier to read

### Actual behaviour

Hard to read

### Detailed description

See screenshot in repro. Message notifications appear as background and hard to read.

### Specifications

Safari 16.0
Mastodon web app

**Comment by simonja7** [comment]
Not the same experience in Light Mode. Notifications are easy to read.

**Comment by afontenot** [comment]
Probably a dupe of https://github.com/mastodon/mastodon/issues/20836

I'm not 100% sure what makes some notifications have OK contrast and some have low.

**Comment by simonja7** [comment]
Yes, a duplicate. And they said it better on 20836. "Contrast" was the word I was looking for! :) The effect is less pronounced in Light view but there is still a drop in contrast.

**Comment by ClearlyClaire** [comment]
Closed as a duplicate of #20836
